tests/h/search/index_test.py

```diff
@@ -77,6 +77,14 @@ def test_it_can_index_an_annotation_with_a_document_with_no_title(self, factorie
 
         assert "title" not in get(annotation.id)["document"]
 
+    def test_it_notifies(self, AnnotationTransformEvent, factories, notify, index, search):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        event = AnnotationTransformEvent.return_value
+        notify.assert_called_once_with(event)
```

Comment:
I think you also need to test that the `AnnotationTransformEvent` was initialized with the right arguments, by testing `AnnotationTransformEvent.assert_called_once_with(...)`